db/seeds.rb

```diff
@@ -1,3 +1,3 @@
 require_relative 'seeds/categories.rb'
 require_relative 'seeds/items.rb'
-
+require_relative 'seeds/jokes.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.